Codeinwp/neve
# fix: search form width when merged with mergeable components [#1278]

### Summary
Fixes issue with search form component and mergeable components.


### Will affect visual aspect of the product

NO

### Test instructions


- Add mergeable components next to the search form component
- Everything should be looking fine


Closes #1278.
